instrumentation/opentelemetry_ecto/test/opentelemetry_ecto_test.exs

```diff
@@ -111,7 +124,7 @@ defmodule OpentelemetryEctoTest do
     attach_handler()
 
     try do
-      Repo.all(from u in "users", select: u.non_existant_field)
+      Repo.all(from(u in "users", select: u.non_existant_field))
```

Comment:
It seems like your formatter configuration isn't honoring the formatter file. Please fix that up as there are a lot of formatting changes here unrelated to your change. Thanks!